src/css/nav.css

```diff
@@ -231,3 +231,12 @@
 .menu__expand-checkbox:checked + label:hover + .menu__expandable {
   opacity: 0.5;
 }
+
+.child-links {
+  display: flex;
+  flex-direction: column;
+}
+
+.child-links a {
+  font-size: 18px;
```

Comment:
I think I'd favor these being a good bit bigger?

For pages that contain children, the children are generally doing a lot of the lifting, and the parent page is generally mostly a TOC and when-you-should-read-these context notes... so I think it would make sense for the pointers to the meatier content to be pretty big.